app/assets/stylesheets/bootstrap_import.scss

```diff
@@ -0,0 +1,143 @@
+@import "bootswatch/flatly/variables";
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "bootswatch/flatly/bootswatch";
+
+$sidebar-hover-bg: darken($navbar-default-bg, 5%);
+
+// from errbit.css
+#content-title {
+  padding-bottom: 25px;
+}
+#content-title h1 {
+  margin-top: 0;
+  padding-bottom: 12px;
+  border-bottom: 1px solid #ccc;
+}
+#content-title .meta { font-size: 0.9em; color: #787878; }
+
+#content-comments a.destroy-comment {
+  color: #EE0000;
+  margin-right: 5px;
+  margin-top: 2px;
+  font-size: 21px;
+  line-height: 1;
+  float: right;
+}
+#content-comments a.destroy-comment:hover {
+  text-decoration: none;
+  color: #AA0000;
+}
+
+/* Footer */
+#footer {
+  padding: 15px 0;
+  font-size: 0.8em; text-align: center;
+  color: #929292;
+}
+
+/* Tables */
+table th, table td { padding: 12px; }
+table th { background-color: darken($gray-lighter, 5%); white-space: nowrap; }
+table .main-column { width: 100%; }
+td.count { text-align: center; }
+
+.field_with_errors {
+  @extend .has-error;
+}
+
+.sidebar {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  display: block;
+  padding: 0 20px;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+#navbar-brand {
+  background: rgba(0, 0, 0, 0) url(images/logo.png) 18px 15px no-repeat;
```

Comment:
This should use the `image-url("…")` Sass helper (otherwise it will 404 in production).